test/error_serialiser_test.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require 'minitest/autorun'
+require 'minitest/mock'
+require_relative '../lib/twiglet/error_serialiser'
+
+describe Twiglet::ErrorSerialiser do
```

Comment:
Apologies if I missed it, but could you add 2 test cases here, one that used to fail previously due to the issue you raised with this PR, and one that would succeed as the normal use case?

This will ensure your changes aren't breaking existing functionality as well as ensuring future changes don't break your currently breaking functionality again.

Reply:
?_?

> This will ensure your changes aren't breaking existing functionality

those tests are in the logger_test.rb

Reply:
that error i was talking about cannot be replicated in twiglet

the problem is the system arent using twiglet, but in tokenhouse it assume it uses twiglet. so if anything that test need to be in tokenhouse @moltenice

Reply:
Thanks for the chat offline :) looks good to me!